docs/user-guide.rst

```diff
@@ -3,36 +3,249 @@
 User Guide
 ==========
 
-Once you have installed Troi, get familiar with how the command line works. To get the usage for Troi, do this:
+Global Playlist generation
+--------------------------
+
+Once you have installed Troi, get familiar with how the command line works. To get the usage for Troi:
 
 .. code-block:: bash
 
-   python -m troi.cli
+   troi --help
 
 
 To list all the patches that are currently available, do:
 
 .. code-block:: bash
 
-   python -m troi.cli list
+   troi list
 
 
-To generate a playlist from a patch and display it in the terminal, do:
+To generate a global (MBID based) playlist from a patch and display it in the terminal, do:
 
 .. code-block:: bash
 
-   python -m troi.cli playlist --print daily-jams <user_name>
+   troi playlist --print daily-jams <user_name>
 
 This will generate a playlist and print the list to the terminal.
 
 
-To generate a playlist from a patch and display it and then upload it to ListenBrainz:
+To generate a global playlist from a patch and display it and then upload it to ListenBrainz:
 
 .. code-block:: bash
 
-   python -m troi.cli playlist --print --upload --token <user-token> daily-jams <user_name>
+   troi playlist --print --upload --token <user-token> daily-jams <user_name>
 
 This will generate a playlist and print the list to the terminal and then upload it to ListenBrainz. You can find your
 user token on your `profile page at ListenBrainz <https://listenbrainz.org/profile/>`_.
 
 From here on you can explore different :ref:`patches` or read how Troi works :ref:`technical-introduction`
+
+
+Local Playlist Generation
+-------------------------
+
+Local playlists are playlists that have been resolved against a local collection
+and are playable to a local player. In order for this to work, you will need
+to index your local collection using the database tools first. 
+
+Index your music collection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**IMPORTANT**: This feature only works if you music collection is tagged with MusicBrainz tags. We recommend
```

Comment:
Is this step needed for collections on funkwhale too? Otherwise probably move the note here, make two subsections one for funkwhale/subsonic and one for local filesystem collections.

Reply:
The step is more related to playlist generation than which service you use. I moved this sentence up higher to be less confusing.